nesc/whip6/apps/ICountRead/ICountReadPrv.nc

```diff
@@ -0,0 +1,42 @@
+/*
+ * whip6: Warsaw High-performance IPv6.
+ *
+ * Copyright (c) 2017 University of Warsaw
+ * All rights reserved.
+ *
+ * This file is distributed under the terms in the attached LICENSE
+ * files.
+ */
+
+#define INTERVAL_1S 1024
+
+module ICountReadPrv {
```

Comment:
Hm, I followed naming convention used in all apps - suffix "App" for configuration and "Prv" for implementation.
Why should I add "AppMain" here?